### DISCUSSION TITLE
How to CDN cache getserversideprops ？

### DISCUSSION THREAD

**Discussion by parlay96** [vercel/next.js#32085] [Help]
### Describe the feature you'd like to request

You should use getServerSideProps only if you need to pre-render a page whose data must be fetched at request time. Time to first byte (TTFB) will be slower than getStaticProps because the server must compute the result on every request, and the result cannot be cached by a CDN without extra configuration.

The official document says that getserversideprops cannot be cached without additional configuration, so what should be the cache

### Describe the solution you'd like

You should use getServerSideProps only if you need to pre-render a page whose data must be fetched at request time. Time to first byte (TTFB) will be slower than getStaticProps because the server must compute the result on every request, and the result cannot be cached by a CDN without extra configuration.

The official document says that getserversideprops cannot be cached without additional configuration, so what should be the cache

### Describe alternatives you've considered

You should use getServerSideProps only if you need to pre-render a page whose data must be fetched at request time. Time to first byte (TTFB) will be slower than getStaticProps because the server must compute the result on every request, and the result cannot be cached by a CDN without extra configuration.

The official document says that getserversideprops cannot be cached without additional configuration, so what should be the cache

**Comment by balazsorban44** [comment]
Hey, you may find [these docs](https://nextjs.org/docs/going-to-production#caching) useful for your use case.

I must warn you that your issue and its description aren't constructive. You have not provided the minimal required detail in accordance with our [support terms](https://vercel.com/legal/support-terms). Future issues in a similar vein will be closed without reply.
